spec/fund_america/api_info_spec.rb

```diff
@@ -2,20 +2,18 @@
 
 describe FundAmerica::ApiInfo do
   context '#info' do
-    before(:all) do
-      @response = FundAmerica::ApiInfo.info
-    end
+    let(:response) { FundAmerica::ApiInfo.info }
```

Comment:
So you are doing real API requests during tests 🤔 I think this is generally a bad practice, I think you should use a tool like VCR.